google-cloud-spanner/lib/google/cloud/spanner/batch_snapshot.rb

```diff
@@ -784,8 +850,8 @@ def self.load data, service: nil, query_options: nil
         ##
         # @private Creates a new BatchSnapshot instance from a
         # `Google::Cloud::Spanner::V1::Transaction`.
```

Comment:
Why is a BatchSnapshot instance getting created from Spanner::V1::Transaction. Here Spanner::V1::Transaction is this a read write transaction?

Reply:
Ideally BatchSnapshot is created from batch client right?
https://github.com/googleapis/ruby-spanner/blob/2659cac84d239c118f018304370b990179d1021e/google-cloud-spanner/lib/google/cloud/spanner/batch_client.rb#L201

Reply:
Huh thats right BatchSnapshot gets created only form BatchClient. This looks like a comment copy pasted while created which is not updated. I will push an update 👍